telethon/client/uploads.py

```diff
@@ -357,7 +361,9 @@ def callback(current, total):
 
         # Not document since it's subject to change.
         # Needed when a Message is passed to send_message and it has media.
-        if 'entities' in kwargs:
+        if formatting_entities is not None:
+            msg_entities = formatting_entities
+        elif 'entities' in kwargs:
```

Comment:
Could the kwargs be removed and just use `formatting_entities` at the call site? (I think kwargs are only there as a "private" entities arg).

Reply:
done, I had assumed it was some `**`-unpack style thing, but turned out it's just a normal `send_file` call